libpod/runtime.go

```diff
@@ -1077,3 +1094,8 @@ func (r *Runtime) ImageRuntime() *image.Runtime {
 func (r *Runtime) SystemContext() *types.SystemContext {
 	return r.imageContext
 }
+
+// GetEventLogFilePath returns the path for the event log file in string form
+func (r *Runtime) GetEventLogFilePath() string {
```

Comment:
Remove Get, just call it EventLogFilePath()

Reply:
I thought we did not want to expose this to anyone - that makes it part of the API, and we need to make guarantees about format? I'd remove this entirely

Reply:
removed ...